src/machine.rs

```diff
@@ -819,34 +820,23 @@ impl OlmMachine {
 
     /// Import the given room keys into our store.
     ///
-    /// `exported_keys` is a list of previously exported keys that should be
-    /// imported into our store. If we already have a better version of a key,
-    /// the key will _not_ be imported.
+    /// Mostly, a deprecated alias for `importExportedRoomKeys`, though the
+    /// return type is different.
```

Comment:
Well, I didn't bother, because it wasn't documented before, and it's deprecated so there shouldn't be any *new* users who would need to know what it was. 

Have added some words to help a bit.